test/extended/openstack/loadbalancers.go

```diff
@@ -84,83 +87,88 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack][lb][Serial] The O
 	for _, i := range availableLbProvidersUnderTests {
 		lbProviderUnderTest := i
 
-		// https://issues.redhat.com/browse/OSASINFRA-2753
-		g.It(fmt.Sprintf("should create an UDP %s LoadBalancer when an UDP svc with type:LoadBalancer is created on Openshift", lbProviderUnderTest), func() {
+		for _, j := range availableProtocolsUnderTests {
+			protocolUnderTest := j
```

Comment:
Why like this? Just assign it directly to `protocolUnderTest`:

```
for _, protocolUnderTest := range availableProtocolsUnderTests {
```

Reply:
That's something we observed in the previous round of tests. Honestly, I don't know why but the whole engine that is calculating the list of tests don't work if I don't do that. I copied the current approach from origin, so I assume they already hit that and that was their solution.

Reply:
right, goroutines in `for` cycles...

To make it a bit more digestible, try this:

```Go
for i := range availableProtocolsUnderTests {
			protocolUnderTest := availableProtocolsUnderTests[i]
```

Reply:
I’d also check if this Ginkgo thing likes being called in a loop.